frontend/src/app/commonComponents/__snapshots__/Pagination.test.tsx.snap

```diff
@@ -75,16 +75,16 @@ exports[`Pagination should render Pagination cases: Pagination '7 pages' 1`] = `
         >
           <svg
             aria-hidden="true"
-            class="svg-inline--fa fa-angle-left fa-w-8 "
```

Comment:
curious if this `fa-w-8` changed the styling for some of these icons?

Reply:
mostly they end up looking slightly different. In this case (new is on top)
![image](https://user-images.githubusercontent.com/10108172/198675529-b365bce1-8dfd-4e5c-b453-e707198756cc.png)
![image](https://user-images.githubusercontent.com/10108172/198675663-7d1c8319-b999-44b1-ae06-a8fe796f1d10.png)